helm/defectdojo/values.yaml

```diff
@@ -23,6 +23,23 @@ repositoryPrefix: defectdojo
 # imagePullSecrets: defectdojoregistrykey
 tag: latest
 
+securityContext:
+  djangoSecurityContext:
+    # django dockerfile sets USER=1001
+    runAsUser: 1001
+  nginxSecurityContext:
+    # nginx dockerfile sets USER=1001
+    runAsUser: 1001
+
+test:
```

Comment:
Can we rename it to:
``` 
tests:
    unit-tests:
```
So we are able to maybe add an integration test or something similar.

Reply:
Done, please take a look